pyproject.toml

```diff
@@ -76,6 +71,9 @@ reportUnneesssaryContains = "warning"
 reportMissingTypeArgument = "error"
 reportUnusedExpression = "warning"
 
+[tool.pyright.defineConstant]
+PYDANTIC_V2 = true
```

Comment:
is this unset somewhere for pydantic-1?

Reply:
Gosh, that's news to me. Thanks for pointing that one out.